aws/resource_aws_emr_cluster_test.go

```diff
@@ -2970,16 +2995,17 @@ resource "aws_emr_cluster" "tf-test-cluster" {
     instance_type  = "c4.large"
   }
 
-  log_uri = "s3://${aws_s3_bucket.test.bucket}/"
```

Comment:
Hi @nikhil-goenka 👋  Thank you for submitting this enhancement. Can you please create a new acceptance test and test configuration to verify this change instead of modifying the existing one? This helps us ensure there are no unexpected regressions. Please see also: https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/contribution-checklists.md#enhancementbugfix-to-a-resource

Reply:
Got the logic, updated